corehq/apps/hqwebapp/static/hqwebapp/js/ui_elements/bootstrap5/ui-element-key-val-list.js

```diff
@@ -25,42 +25,49 @@ hqDefine('hqwebapp/js/ui_elements/bootstrap5/ui-element-key-val-list', [
         this.sub_title = subTitle ? '<p>' + subTitle + '</p>' : '';
         this.max_display = maxDisplay;
 
-        this.$edit_view = $('<div class="well well-sm" />');
-        this.$edit_view.appendTo(this.ui);
+        this.$edit_view_wrapper = $('<div class="card mb-2"></div>');
+        this.$edit_view = $('<div class="card-body"></div>');
+        this.$edit_view.appendTo(this.$edit_view_wrapper);
+        this.$edit_view_wrapper.appendTo(this.ui);
 
         this.$formatted_view = $('<input type="hidden" />');
         this.$formatted_view.appendTo(this.ui);
 
-        this.$modal_trigger = $('<a class="btn btn-default enum-edit" href="#' + this.modal_id + '" ' +
-            'data-toggle="modal" />').html('<i class="fa fa-pencil"></i> ' + gettext('Edit'));
+        this.$modal_trigger = $('<a class="btn btn-outline-primary enum-edit" href="#' + this.modal_id + '" ' +
+            'data-bs-toggle="modal" />').html('<i class="fa fa-pencil"></i> ' + gettext('Edit'));
 
         // Create new modal controller for this element
         var $enumModal = $('<div id="' + this.modal_id + '" class="modal fade hq-enum-modal" />');
         var $modalDialog = $('<div class="modal-dialog"/>');
         var $modalContent = $('<div class="modal-content" />');
 
-        $modalContent.prepend('<div class="modal-header"><a class="close" data-dismiss="modal">×</a><h4 class="modal-title">'
-            + this.modal_title + '</h4>' + this.sub_title + '</div>');
-        var $modalForm = $('<form class="form-horizontal hq-enum-editor" action="" />'),
+        $modalContent.prepend('<div class="modal-header">'
+            + '<h4 class="modal-title">'
```

Comment:
i know this was here before, but i always get uneasy when I see html inside of javascript.... Always prefer a template reference approach...